sdk/communication/Azure.Communication.Sms/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2021-03-09)
```

Comment:
Does SMS have a 2021-03-09 release date as well? I thought SMS is being released later.